api/routes/subcriterias/get_sub_criteria.py

```diff
@@ -74,6 +74,21 @@ def get_themes_fields(
         return f"Incorrect theme id -> {theme_id}"
 
 
+def get_all_uploaded_document_field_answers(
```

Comment:
can we filter these answers with application_id also? i.e. combine the function here with the 'get_all_uploaded_document_theme_answers()' function?

Reply:
Hey - not sure what you mean by filter by application id? I think at this point we only have data specific to a single application!

Reply:
perfect!